.github/workflows/publish-tasks.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish to DockerHub
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+env:
+  DOCKER_USER: bilts
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Log into registry
+        run: echo "${{ secrets.DOCKERHUB }}" | docker login -u "${DOCKER_USER}" --password-stdin
+      - name: Setup Node
+        run: |
+          npm install -g npm@latest
```

Comment:
I started changing this to `npm install -g npm@7` to explicitly make sure we use version 7.

Reply:
Done.  Also filed https://bugs.earthdata.nasa.gov/browse/HARMONY-842 to make us stop needing to do that